layer/VkLayer_khronos_profiles.json.in

```diff
@@ -305,12 +305,32 @@
                                         {
                                             "key": "DEFAULT_FEATURE_VALUES_DEVICE",
                                             "label": "Use Device Values",
-                                            "description": "When a feature is not mentionned in the select Vulkan profiles, set is to the physical device value."
+                                            "description": "When a feature is not mentioned in the select Vulkan profiles, set is to the physical device value."
                                         },
                                         {
                                             "key": "DEFAULT_FEATURE_VALUES_FALSE",
                                             "label": "Use False",
-                                            "description": "When a feature is not mentionned in the select Vulkan profiles, set it to false."
+                                            "description": "When a feature is not mentioned in the select Vulkan profiles, set it to false."
+                                        }
+                                    ]
+                                },
+                                {
+                                    "key": "unknown_feature_values",
+                                    "label": "Unknown Features",
```

Comment:
Maybe "Allow Unknown Features" ?

Reply:
Unknown features are always allowed, the question is just if you attempt to load them from the driver or ignore them